editor/editor_node.cpp

```diff
@@ -7361,7 +7361,7 @@ EditorNode::EditorNode() {
 	vcs_actions_menu = VersionControlEditorPlugin::get_singleton()->get_version_control_actions_panel();
 	vcs_actions_menu->set_name("Version Control");
 	vcs_actions_menu->connect("index_pressed", callable_mp(this, &EditorNode::_version_control_menu_option));
-	vcs_actions_menu->add_item(TTR("Create Version Control Metadata..."), RUN_VCS_METADATA);
+	vcs_actions_menu->add_item(TTR("Create/Override Version Control Settings..."), RUN_VCS_METADATA);
```

Comment:
Note that the next item also contains "Settings":
![](https://github.com/godotengine/godot/assets/47700418/19517adb-3939-4bef-a215-16329e849bba)

Reply:
Ah must've been a mistake, I didn't mean to write that. Will fix.